org.fritzing.Fritzing.json

```diff
@@ -72,7 +72,8 @@
                 {
                     "type": "patch",
                     "paths": [
-                        "patches/fritzing-appdata-screenshots.patch"
+                        "patches/fritzing-appdata-screenshots.patch",
+                        "patcheds/add-wm-class.patch"
```

Comment:
A typo?
```suggestion
                        "patches/add-wm-class.patch"
```

Reply:
Whoops, yes that was a typo. I just fixed it in my branch.